great_expectations/expectations/metrics/column_aggregate_metrics/column_quantile_values.py

```diff
@@ -329,6 +340,25 @@ def _get_column_quantiles_sqlite(
         raise pe
 
 
+def _get_column_quantiles_athena(
+    column,
+    quantiles: Iterable,
+    allow_relative_error: bool,
+    dialect,
+    selectable,
+    sqlalchemy_engine,
+) -> list:
+
+    approx_percentiles = f"approx_percentile({column}, ARRAY{list(quantiles)})"
+    selects_approx: List[TextClause] = [sa.text(approx_percentiles)]
+    quantiles_query_approx: Select = sa.select(selects_approx).select_from(selectable)
+    quantiles_results: Row = sqlalchemy_engine.execute(
```

Comment:
Can you wrap your call to `sqlalchemy_engine.execute` in a try/except block to mimic similar funcs?

```
    try:
        quantiles_results: Row = sqlalchemy_engine.execute(...)
        return list(quantiles_results)[0]
    except ProgrammingError as pe:
        exception_message: str = "An SQL syntax Exception occurred."
        exception_traceback: str = traceback.format_exc()
        exception_message += (
            f'{type(pe).__name__}: "{str(pe)}".  Traceback: "{exception_traceback}".'
        )
        logger.error(exception_message)
        raise pe
```

Also shouldn't need the `ast.literal_eval(...)`

Reply:
Thanks a lot @kenwade4 ,  applied the changes and an explanation why ast.literal is needed.  If there is a better way, please let me know and I will apply the needed changes 👍 .